CHANGES.md

```diff
@@ -74,6 +74,7 @@
 ## Deprecations
 
 * X behavior is deprecated and will be removed in X versions ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
+* RedisIO.readAll() was removed since it was deprecated since Beam 2.13.0 ([BEAM-12214](https://issues.apache.org/jira/browse/BEAM-12214)).
```

Comment:
RedisIO.readAll() is removed now. It was deprecated in Beam 2.13.0 ... Please use RedisIO.readKeyPatterns() for the equivalent functionality.